src/test/java/com/synopsys/integration/jira/common/rest/oauth1a/AuthenticateOAuthTest.java

```diff
@@ -59,60 +54,91 @@ public void createAuthUrl() throws Exception {
         URL url = new URL(authorizationUrl);
         assertTrue(StringUtils.isNotBlank(url.toString()));
 
-        System.out.println(authorizationToken);
-        System.out.println(authorizationUrl);
+        System.out.println("Please copy this temporary token into the authCredentials: " + authorizationToken);
+        System.out.println("Please follow this link and copy the verification code into the authCredentials: " + authorizationUrl);
+    }
+
+    // Set the two printed values here
+    private void setTemporaryTokenAndVerificationCode() throws Exception {
```

Comment:
Should this be moved to the top of the test class so other user's know to set these values if they want to run these tests?